docs/cristin.ttl

```diff
@@ -68,31 +88,54 @@
 :varbeidhovedkatkode
     a            rdf:Property ;
     rdfs:comment """
-    The main category of the resource,
-    The range of this property is found here: https://api.cristin.no/v2/results/categories?lang=nb,en
+    The general category of the resource. A collection of "varbeidunderkatkode".
     """ ;
     rdfs:range   xsd:string ;
     rdfs:domain  :Result .
 
 :varbeidunderkatkode
     a            rdf:Property ;
-    rdfs:comment "The secondary category of the resource. No specific range available at the moment." ;
+    rdfs:comment """
+        The more specific category of the resource. No specific range available at the moment.
```

Comment:
I can export you the 70-ish primary db keys if you want?

Reply:
please do so.